### DISCUSSION TITLE
new user

### DISCUSSION THREAD

**Discussion by raza4502** [nehal2210/Blockchain-Engineer-Course#15] [Q&A] [answered]
sir men bilkul in chezon men new hun or isko achy sy sekhna chahta hun 1st time development men aya hun mujhy kahan sy start lena hoga men apki block chain ki 9 videos dekh chuka hun lakin just 10% smjh aya

**Comment by nehal2210** [comment]
If you did not have any experience in programming then you just have to learn javascript first. In youtube, you can follow any tutorial to learn it and also do practice it on Hackerrank.

**Comment by raza4502** [comment]
Sir kindly mention recommended links of videos

**Reply by nehal2210** [reply, marked as answer]
This course is currently live, maybe it will be helpful
https://www.youtube.com/@IshaqBhojani/playlists
mark as answer if you found it useful.

**Comment by raza4502** [comment]
Thank you, I will check it out.
